src/utils/dateTimeFormatter.test.ts

```diff
@@ -15,3 +19,21 @@ describe("formatDate", () => {
     expect(formatDate(testDate)).toBe("25 Aug 2020");
   });
 });
+
+describe("formatGovWalletDateTimeToSallyTimestamp", () => {
+  it("should return the correct date format", () => {
```

Comment:
what happens if the time zone isn't GMT+8?

Reply:
since the timezone is controlled on the backend, should be fine for us to handle GMT+8!